packages/shopware-6-client/__tests__/services/CustomerService/getCustomerAddress.spec.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  getCustomerAddress,
+  update,
+  config
+} from "@shopware-pwa/shopware-6-client";
+import { getCustomerAddressEndpoint } from "../../../src/endpoints";
+import { apiService } from "../../../src/apiService";
+import { random } from "faker";
+
+jest.mock("../../../src/apiService");
+const mockedAxios = apiService as jest.Mocked<typeof apiService>;
+
+describe("CustomerService - getCustomerAddress", () => {
+  let contextToken: string;
+  beforeEach(() => {
+    jest.resetAllMocks();
+
+    contextToken = random.uuid();
+
+    update({ contextToken });
+  });
+  afterEach(() => {
+    // expect(
+    //   mockedAxios.defaults.headers.common["sw-context-token"]
+    // ).toEqual(contextToken);
```

Comment:
We may delete that comment. : ) Probably it is one of the tested scenarios during remote pair programming, which is no longer needed. : )

Reply:
removed